dao/src/main/java/org/thingsboard/server/dao/notification/DefaultNotifications.java

```diff
@@ -372,6 +374,20 @@ public class DefaultNotifications {
                     .build())
             .build();
 
+    public static final DefaultNotification resourcesShortage = DefaultNotification.builder()
+            .name("Resources shortage notification")
+            .type(NotificationType.RESOURCES_SHORTAGE)
+            .subject("Warning: ${resource} shortage")
+            .text("${resource} usage is at ${usage}%.")
+            .icon("warning")
+            .rule(DefaultRule.builder()
+                    .name("Resources shortage")
+                    .triggerConfig(ResourcesShortageNotificationRuleTriggerConfig.builder().cpuThreshold(0.8f).storageThreshold(0.8f).ramThreshold(0.8f).build())
+                    .description("Send notification to system admins when resources shortage is running low")
```

Comment:
shortage can't run low.
Send notification to system admins on resource shortage